mvp-spec/assets/generated-transaction-nominal-case.svg

```diff
@@ -1 +1 @@
y="198" class="messageText" style="text-anchor: middle;">Return an OpenRTB bid response</text><line x1="675" y1="205" x2="475" y2="205" class="messageLine0" stroke-width="2" stroke="black" marker-end="url(#arrowhead)" style="fill: none;"></line></g><g><text x="675" y="233" class="messageText" style="text-anchor: middle;">Send an OpenRTB bid request</text><line x1="475" y1="240" x2="875" y2="240" class="messageLine0" stroke-width="2" stroke="black" marker-end="url(#arrowhead)" style="fill: none;"></line></g><g><text x="675" y="268" class="messageText" style="text-anchor: middle;">Return an OpenRTB bid response</text><line x1="875" y1="275" x2="475" y2="275" class="messageLine0" stroke-width="2" stroke="black" marker-end="url(#arrowhead)" style="fill: none;"></line></g><g><text x="475" y="303" class="messageText" style="text-anchor: middle;">Select a winner to the bid request</text><path d="M 475,310 C 535,300 535,340 475,330" class="messageLine0" stroke-width="2" stroke="black" marker-end="url(#arrowhead)" style="fill: none;"></path></g><g><text x="375" y="368" class="messageText" style="text-anchor: middle;">Return data to display the ad</text><line x1="475" y1="375" x2="275" y2="375" class="messageLine0" stroke-width="2" stroke="black" marker-end="url(#arrowhead)" style="fill: none;"></line></g><g><text x="175" y="403" class="messageText" style="text-anchor: middle;">Display the webpage with the ad.</text><line x1="275" y1="410" x2="75" y2="410" class="messageLine0" stroke-width="2" stroke="black" marker-end="url(#arrowhead)" style="fill: none;"></line></g><g><rect x="0" y="430" fill="#eaeaea" stroke="#666" width="150" height="65" rx="3" ry="3" class="actor"></rect><text x="75" y="462.5" dominant-baseline="central" alignment-baseline="central" class="actor" style="text-anchor: middle;"><tspan x="75" dy="0">User</tspan></text></g><g><rect x="200" y="430" fill="#eaeaea" stroke="#666" width="150" height="65" rx="3" ry="3" class="actor"></rect><text x="275" y="462.5" dominant-baseline="central" alignment-baseline="central" class="actor" style="text-anchor: middle;"><tspan x="275" dy="0">Publisher</tspan></text></g><g><rect x="400" y="430" fill="#eaeaea" stroke="#666" width="150" height="65" rx="3" ry="3" class="actor"></rect><text x="475" y="462.5" dominant-baseline="central" alignment-baseline="central" class="actor" style="text-anchor: middle;"><tspan x="475" dy="0">SSP</tspan></text></g><g><rect x="600" y="430" fill="#eaeaea" stroke="#666" width="150" height="65" rx="3" ry="3" class="actor"></rect><text x="675" y="462.5" dominant-baseline="central" alignment-baseline="central" class="actor" style="text-anchor: middle;"><tspan x="675" dy="0">DSP1</tspan></text></g><g><rect x="800" y="430" fill="#eaeaea" stroke="#666" width="150" height="65" rx="3" ry="3" class="actor"></rect><text x="875" y="462.5" dominant-baseline="central" alignment-baseline="central" class="actor" style="text-anchor: middle;"><tspan x="875" dy="0">DSP2</tspan></text></g></svg>
```

Comment:
If this commit doesn't fix the problem of comparison between generated svgs, I suggest to dismiss this file.

Reply:
you're right (unfortunately 😭)
I will revert this